install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -5,6 +5,24 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Return fqdn cilium image
```

Comment:
As I mentioned in https://github.com/cilium/cilium/pull/16900#discussion_r672166934, this comment is confusing...